gusb/gusb-device-event.c

```diff
@@ -100,6 +102,10 @@ _g_usb_device_event_save(GUsbDeviceEvent *self, JsonBuilder *json_builder, GErro
 		json_builder_set_member_name(json_builder, "Status");
 		json_builder_add_int_value(json_builder, self->status);
 	}
+	if (self->rc != LIBUSB_SUCCESS) {
+		json_builder_set_member_name(json_builder, "Error");
```

Comment:
do you want the error to be an object? that way it would be extendable in the future, to i.e have a description

Reply:
Maybe the name that's made it confusing? It's really just the raw error code from libusb, i.e. `-4` for `LIBUSB_ERROR_NO_DEVICE`

Reply:
I see... dunno then

Reply:
Maybe it should be called `ErrorNumber`, `ErrorCode` or `Rc` or something?

Reply:
libusb just calls it `libusb_error`